.github/workflows/main.yml

```diff
@@ -25,7 +25,7 @@ jobs:
           coverage: "--cov ./ --cov-append --cov-config setup.cfg"
         - name: py310-dev
           python: 3.10-dev
-          docutils: du16
+          docutils: du17
```

Comment:
After https://github.com/sphinx-doc/sphinx/pull/9071 merged, this spec does not work well.